src/model/Item.js

```diff
@@ -1,11 +1,12 @@
 class Item {
   /**
-   * @param {String} id
-   * @param {Product} product
-   * @param {Date} date
-   * @param {Unity} unity
-   * @param {Number} quantity
-   * @param {Boolean} active
+   * Item details
```

Comment:
- this should be a typedef
- and use `http://usejsdoc.org/tags-property.html` to define the properties

Reply:
Fix commit ddee944a78397c5139762b1cb49167434b0b7cf5